home/.vim/config/bundles

```diff
@@ -25,6 +25,8 @@ Bundle 'editorconfig/editorconfig-vim'
 Bundle 'bogado/file-line'
 Bundle 'jceb/vim-orgmode'
 Bundle 'tpope/vim-speeddating'
+Bundle 'tpope/vim-obsession'
+Bundle 'dhruvasagar/vim-prosession'
```

Comment:
This creates a file cache allowing you to reboot, etc. and have everything restored. Also, if vim bugs out on you and you have to kill it, this would restore the session on re-entry.